### PR TITLE
Support for arm64-v8a on Android

### PR DESCRIPTION
**Pull request by superjohan**
Adds support for 64-bit ARM on Android. The end result seems to work in my apps at least. Not sure if this is done in a way that's proper for the project; maybe someone can help me out if not.

I tried getting x86_64 on Android working as well, but with less luck.

### CODE REVIEW

**Comment by 44xiao44** [comment]
I want to use `.so package` that supports arm64. Can I download Maven Central v4.6.0?

**Comment by superjohan** [comment]
You need to build it yourself, unfortunately. I also had trouble getting a proper x86 build going, so I did a frankenbuild.

**Comment by wwqgtxx** [comment]
Could provide a precompiled `libj2v8.so` arm64-v8a lib file before owner pull this pr to help more people need use j2v8 with 64bit library

**Comment by wwqgtxx** [comment]
I had compiled a arm64-v8a `libj2v8.so` by this branch [libj2v8.so](https://raw.githubusercontent.com/wwqgtxx/AndroidCalculator/master/app/src/main/jniLibs/arm64-v8a/libj2v8.so), hope can help the people who need.

**Comment by superjohan** [comment]
Turns out this PR is incomplete in other ways too: the armv7 build doesn't work on pre-Lollipop devices. Reverting https://github.com/eclipsesource/J2V8/pull/380/commits/7d9c4564c0dbac60558a66d718a9a5473b1aa9f2 "fixes" it, but that means arm64 builds stop working. I'll see if I can get something done about this at some point.

**Comment by wwqgtxx** [comment]
I found this problem too and now use the official armv7 so file to "solve".Hope can fix in future.

**Comment by jordanffs** [comment]
Is this stalled? Would love to see this merged at some point soon.

File: build_system/config_android.py [hunk 1]
```diff
@@ -21,13 +21,14 @@
...
-    c.arch_x86: "x86"
+    c.arch_x86: "x86",
+    c.arch_arm64: "arm64-v8a"
 })
 
 #-----------------------------------------------------------------------
 def build_node_js(config):
     return [
-        """android-gcc-toolchain $ARCH --api 17 --host gcc-lpthread -C \
+        """android-gcc-toolchain $ARCH --api 21 --host gcc-lpthread -C \
```
Comment (gengjiawen) [inline build_system/config_android.py]: Only need to set api to 21 when build arm64, maybe return different config for different abi.

**Comment by superjohan** [comment]
Seems this is no longer needed.

**Pull request closed without merging**